CMakeLists.txt

```diff
@@ -54,7 +54,7 @@ option(EPROSIMA_BUILD_TESTS "Activate the building tests" OFF)
 option(THIRDPARTY "Activate the build of thirdparties" OFF)
 option(VERBOSE_SERIALIZATION "Use verbose output" OFF)
 option(VERBOSE_MESSAGE "Use verbose output" OFF)
-if((${CMAKE_SYSTEM_NAME} STREQUAL "Linux") OR (${CMAKE_SYSTEM_NAME} STREQUAL "Generic"))
+if(PLATFORM_NAME_LINUX)
```

Comment:
This should be the opposite `NOT PLATFORM_NAME_LINUX`

Reply:
No, it enables the option `MEMORY_PERFORMANCE` for Linux, that is right.
The problem is the `if` case with `PLATFORM_NAME_LINUX` variable which is no set at that point. It is fixed!